teps/0079-tekton-catalog-support-tiers.md

```diff
@@ -131,194 +134,163 @@ surfacing the verification information and building a corpus of verified resourc
-   the SLO is one week. The automated testing is discussed further
-   [below](#automated-testing-and-dogfooding-1).
-4. The resource MUST be scanned for common vulnerabilities and exposures.
-   If any issues are discovered, they MUST be patched or disclosed as soon as
-   possible; the SLO is one week. The scanning for CVEs is discussed further
-   [below](#image-scanning-for-common-vulnerabilities-and-exposures-cves-1).
-5. The resource MUST support verification as proposed in
-   [TEP-0091: Verified Remote Resources][tep-0091]. For now, it MUST be
-   published to a public OCI registry as a [Tekton Bundle][bundle] and
-   signed by Tekton. In the future, it MUST be updated to support accepted
-   proposal of TEP-0091 depending on the direction of
-   [TEP-0060: Remote Resource Resolution][tep-0060].
-   The verification is discussed further [below](#verified-remote-resources-1).
-6. The resource MUST be well documented with all configuration options described
-   and a working example provided.
-7. The resource MUST follow and demonstrate best practices, such as the
-   [`Task` authoring recommendations][task-authoring-recommendations].
-   If there are new best practices, the resource MUST be updated if needed.
-8. The resource SHOULD be updated to the latest version of Tekton and other
-   dependencies.
-
-#### Promotion from Community Catalog to Official Catalog
-
-Promoting a resource from the Community Catalog to the Official Catalog 
-should be an exception, not the norm. The set of resources in the 
-Official Catalog should relatively small so that it's sustainable for the 
-Tekton Catalog Owners to maintain. 
-
-A resource in the Community Catalog may get promoted to the Official Catalog 
-if:
+These are requirements for Official Catalogs:
+
+1. The Catalog MUST comply with the contract and versioning defined in [TEP-0115][tep-0115].
+2. The Catalog MUST define an OWNER file listing the Tekton Catalog Maintainers. The ownership responsibilities are
+described [below](#responsibilities).
+3. The Catalog MUST have automated testing using Tekton for dogfooding. If there are any failures, they MUST be
+resolved as soon as possible; the SLO is one week. Automated testing is discussed further
+[below](#automated-testing-and-dogfooding-1).
+4. The Catalog MUST be scanned for common vulnerabilities and exposures. If any issues are discovered, they MUST be
+patched or disclosed as soon as possible; the SLO is one week. Scanning for CVEs is discussed further
+[below](#image-scanning-for-common-vulnerabilities-and-exposures-cves-1).
+5. The Catalog MUST support verification as proposed in [TEP-0091: Verified Remote Resources][tep-0091]. For now, 
+it MUST be published to a public OCI registry as a [Tekton Bundle][bundle] and signed by Tekton. In the future, it MUST
+be updated to support accepted proposal of [TEP-0091: Verified Remote Resources][tep-0091]. Verification is discussed
+further [below](#verified-remote-resources-1).
```

Comment:
The referred section is not added, are we going to add it?

My question is how will the catalog expose the public keys information?
Suppose user A submit the signed resource, how does user B know where to get the public key to verify it when using the resource?